rfcs/0000-occupied-capacity-increment-rate/0000-occupied-capacity-increment-rate.md

```diff
@@ -0,0 +1,57 @@
+---
+Number: 0000
+Category: Standards Track
+Status: Proposal
+Author: Jinyang Jiang
+Organization: Nervos Foundation
+Created: 2019-2-13
+---
+
+# Occupied capacity increment rate
+
+## Abstract
+
+Capacity is one of the core resources in CKB network, itself is just a number representing how much storage resource a user can use, once a user store something in CKB, the occupied part of storage which we called occupied capacity represents the actually used resource in the network.
+
+Occupied capacity affects CKB network because that every full node must cost the same memory or disk to save these content, it incentives miners to be more centralized which does not make any sense for a decentralized network.
+
+So the solution is to limit the increase occupied capacity in each block, to indirectly control the network's total occupied capacity and the increment rate of occupied capacity.
+
```

Comment:
You have the reason, but what's the target of the limit policy, match the current level of hardware?
Maybe we should supplement this part.

Reply:
This part should be introduced in an RFC that describes on-chain governance mechanics.